pkg/provider/acme/provider.go

```diff
@@ -340,6 +357,62 @@ func (p *Provider) getClient() (*lego.Client, error) {
 	return p.client, nil
 }
 
+func (p *Provider) createHTTPClient() (*http.Client, error) {
+	tlsConfig, err := p.createClientTLSConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	return &http.Client{
+		Timeout: 2 * time.Minute,
+		Transport: &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
+			DialContext: (&net.Dialer{
+				Timeout:   30 * time.Second,
+				KeepAlive: 30 * time.Second,
+			}).DialContext,
+			TLSHandshakeTimeout:   30 * time.Second,
+			ResponseHeaderTimeout: 30 * time.Second,
+			TLSClientConfig:       tlsConfig,
+		},
+	}, nil
+}
+
+func (p *Provider) createClientTLSConfig() (*tls.Config, error) {
+	if len(p.CACertificates) > 0 {
+		certPool, err := lego.CreateCertPool(p.CACertificates, p.CAUseSystemCertPool)
```

Comment:
@ldez WDYT of using of leveraging also `LEGO_CA_SYSTEM_CERT_POOL` env variable here?

Reply:
It's better to use env vars only with the global configuration: mixing configuration and env vars will create confusion.